plugins/woocommerce-admin/client/products/fields/attribute-term-input-field/attribute-term-input-field.tsx

```diff
@@ -0,0 +1,172 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { CheckboxControl, Spinner } from '@wordpress/components';
+import { resolveSelect } from '@wordpress/data';
+import { useCallback, useEffect, useState } from '@wordpress/element';
+import { useDebounce } from '@wordpress/compose';
+import {
+	EXPERIMENTAL_PRODUCT_ATTRIBUTE_TERMS_STORE_NAME,
+	ProductAttributeTerm,
+} from '@woocommerce/data';
+import {
+	selectControlStateChangeTypes,
+	__experimentalSelectControl as SelectControl,
+	__experimentalSelectControlMenu as Menu,
+	__experimentalSelectControlMenuItem as MenuItem,
+} from '@woocommerce/components';
+
+type AttributeTermInputFieldProps = {
+	value?: ProductAttributeTerm[];
```

Comment:
I've noticed we're using the generic prop name `value` in a number of these components. Would it make sense to make it something more descriptive/explicit? If I'm working in several of them, it can become a bit confusing.

Reply:
The reason for doing this, is that I wanted to follow the same format as an InputField (envisioning that every field has a `value` prop and a `onChange` handler. 
This way, if we go the schema route for example it will be much easier to integrate these field components within a form and we don't have to add a custom alias for each one.

I know this has mostly been something I have been adhering to, but it hasn't really been a team discussion, I could mention this in Slack if you think that is valuable, depending on what you think of this approach?

Reply:
@louwie17 When you say "going the schema route," are you talking in terms of documentation? I'm not all too familiar with how that would look; it sounds like an alias within the component (while keeping the `value` prop name) would still influence the schema in this way?

That being the case, sounds good to me. Although this should definitely be something that we make the team aware of, since it would require consistency in order to be more valuable.